grails-app/conf/application.yml

```
@@ -153,7 +153,7 @@ grails:
         springsecurity:
             successHandler:
                 alwaysUseDefault: true
-                defaultTargetUrl: "/"
+                defaultTargetUrl: "/customer/dashboard"
```

só dashboard, já não serve? 
```suggestion
                defaultTargetUrl: "/dashboard"
```

---

me parece que compensar deixar separado, quando vemos dashboard já da a entender que precisa estar logado, então não precisaria dessa relação direta com customer... sugiro também uma controller separada só pra dashboard, pra não deixar fora de contexto